### ISSUE TITLE
Address valid attributes

### ISSUE THREAD

**Issue by bingalls** [geocoder-php/Geocoder#539]
As a user of Geocoder/Model/Address, I expect calls to getCountryCode() to return null or empty string, rather than an error. In particular,
`return $this->country->getCode();`
might have _country_ defined as **null** which is the constructor's default.
This is also a problem in toArray().
Further, there is no getCountryName(), so you should likely create that, else mark this method as @deprecated.

To recap, the legitimate code fragment
`$address = new Address();
var_export( $address->toArray() );`
throws an error, rather than print out
`['latitude' => null, 'adminlevels' => [], ...]`

**Comment by Nyholm** [comment]
Thank you for this issue. Have a look at the new `Position` interface in #529. That addresses your issue. I even added a test for that.

**Comment by bingalls** [comment]
Hmm. I like what I'm reading in the discussion. I can review your code, when it arrives, if this is what you mean.

**Comment by Nyholm** [comment]
You may review the code now if you like. See if it would solve your issue and give it a 👍  or 👎 .

**Comment by bingalls** [comment]
Sorry, I could not find the code, outside of the discussion.

**Comment by Nyholm** [comment]
The script:
```php
<?php

include "vendor/autoload.php";

$address = new \Geocoder\Model\Address(); 
var_export( $address->toArray() );
```

Will produce: 

```
% php test.php
array (
  'latitude' => NULL,
  'longitude' => NULL,
  'bounds' => 
  array (
    'south' => NULL,
    'west' => NULL,
    'north' => NULL,
    'east' => NULL,
  ),
  'streetNumber' => NULL,
  'streetName' => NULL,
  'postalCode' => NULL,
  'locality' => NULL,
  'subLocality' => NULL,
  'adminLevels' => 
  array (
  ),
  'country' => NULL,
  'countryCode' => NULL,
  'timezone' => NULL,
) 
```

Which is as you expect. This issue is solved by #529
